.github/workflows/e2e-jimm-k8s-charm-oidc-auth.yml

```diff
@@ -63,12 +63,15 @@ jobs:
           DASHBOARD_ADDRESS=http://$(juju show-unit dashboard/0 | yq .dashboard/0.address):8080
           juju config jimm cors-allowed-origins="$DASHBOARD_ADDRESS"
           juju config jimm juju-dashboard-location="$DASHBOARD_ADDRESS"/models
+      - name: Set the CLI admin's password
+        run: echo ${{ inputs.admin-password }} | juju change-user-password --no-prompt
```

Comment:
We need to do this so we can log out of the admin account without losing access.